sql/core/src/test/scala/org/apache/spark/sql/execution/datasources/json/JsonSuite.scala

```diff
@@ -969,6 +969,14 @@ class JsonSuite extends QueryTest with SharedSQLContext with TestJsonData {
       withTempTable("jsonTable") {
         val jsonDF = sqlContext.read.json(corruptRecords)
         jsonDF.registerTempTable("jsonTable")
+        val jsonDFWithDropMalformed =
+          sqlContext.read.option("mode", "DROPMALFORMED").json(corruptRecords)
+        jsonDFWithDropMalformed.registerTempTable("jsonTableWithDropMalformed")
```

Comment:
Will we persistent JSON options along with data when register table? If not, I think this `jsonTableWithDropMalformed` is identical to `jsonTable`.